acceptance-tests/values.yaml

```diff
@@ -58,3 +58,13 @@ configuration:
     localAccount:
       username: myusername
       password: mypassword
+
+# If enabled, this will create a PodMonitor object in order
+# to gather metrics on the tests, as allure provides a prometheus compatible
+# file which can be scrapped.
+podMonitor:
+  enabled: false
+  matchLabels:
+    app.camptocamp.com/name: apache-nas
```

Comment:
since https://github.com/camptocamp/charts-gs/commit/0d796c327ea99760566742677cb751c6e0b8e4dc

this is now `app.kubernetes.io/name=apache-nas`

Reply:
yes, I just replaced inrae.fr by camptocamp, but I'm fine with your suggestion.